app/models/case.rb

```diff
@@ -21,8 +22,7 @@ def self.parse_repo_page(response)
                           when 'No'
                             false
                           end
-    item[:internal_bay_small] = get_int('Internal 2.5" Drive Bays')
-    item[:internal_bay_large] = get_int('Internal 3.5" Drive Bays')
+    item[:internal_bays] = { 'bay2.5' => get_int('Internal 2.5" Drive Bays'), 'bay3.5' => get_int('Internal 3.5" Drive Bays') }
```

Comment:
I modified this since on my local, these were the missing PR that I wasn't able to pull